ubuntu2004_cuda/Dockerfile

```diff
@@ -50,7 +50,7 @@ ENV GET curl --location --silent --create-dirs
 ENV UNPACK_TO_SRC tar -xz --strip-components=1 --directory src
 ENV PREFIX /usr/local
 
-# CMake 3.17.3 version in APT is too old
+# CMake 3.16.3 version in APT is too old
 # requires rsync; installation uses `rsync` instead of `install`
 RUN mkdir src \
   && ${GET} https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-Linux-x86_64.tar.gz \
```

Comment:
I'm confused. :confused: Is this branch not up to date with the latest version of the repository? How come it's not CMake 3.21.2 being downloaded here? :confused:

Reply:
Up to date now.